src/System.Configuration.ConfigurationManager/tests/System/Configuration/StringValidatorTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Configuration;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class StringValidatorTests
+    {
+        [Fact]
+        public void Constructor_MinLength()
+        {
+            //We don't expect this to fail
```

Comment:
Any way to test these a bit more as well? A test without an assertion always bothers me unless it can be justified (e.g no applicable API to get the values)